doc/getting-started.md

```diff
@@ -172,3 +180,37 @@ The default font for material design is [`Roboto`](https://fonts.google.com/spec
 	<MaterialFonts xmlns="using:Uno.Material"
 				   OverrideSource="ms-appx:///Style/Application/MaterialFontsOverride.xaml" />
 	```
+
+## Uno.Cupertino
+Uno Cupertino is an add-on package that lets you apply [Cupertino - Human Interface Guideline styling](https://developer.apple.com/design/human-interface-guidelines) to your application with a few lines of code. 
+
+It features:
+- Color system for both Light and Dark themes
+- Styles for existing WinUI controls like Buttons, TextBox, etc.
+
+### Getting Started
+1. Add NuGet package `Uno.Cupertino` to each of project heads
```

Comment:
Maybe mention that there is a Uno.Cupertino and a Uno.Cupertino.WinUI package and when to use which. Similar to the Material section further up on this page.

Reply:
Thanks for the feedback @kazo0 will make the proper changes

Reply:
Done